### ISSUE TITLE
目前生成的角色脸部相对稳定，但是服装不稳定

### ISSUE THREAD

**Issue by xiehuiqi220** [HVision-NKU/StoryDiffusion#157]
总体来说还是不错的，脸部和发型基本稳定，但是服装会出现颜色不一致的情况，希望后面能解决；
顺带提个建议，多角色一致已经是一个世界难题了，希望大佬们精心研究这个，至于生成连环画和视频，我觉得图片排版为连环画不是个什么难题，至于生成视频，事实上又是另外一个世界级课题了，希望大佬们专注于这个项目解决角色持久性问题，不要分散精力。

**Comment by gsiisg** [comment]
Yes the facial characteristics seems to persist but the clothing and environment are still randomly generated even with the same prompt.  I guess the next feature instead of facial consistency is character face+cloth+environment consistency.
